### ISSUE TITLE
Ability to download conflict results as images

### ISSUE THREAD

**Issue by josephwb** [OpenTreeOfLife/opentree#998]
It would be nice to be able to download images of the results of the conflict analyses (say, [this](https://tree.opentreeoflife.org/curator/study/view/ot_412/?tab=trees&tree=Tr77157&conflict=synth)). At the moment it is not possible to even use a screen grab, as most trees are so large that they involve scrolling to see the entire thing:

![untitled](https://cloud.githubusercontent.com/assets/808499/16710879/2938333e-460e-11e6-83aa-c21f03640259.png)

For example, I can see people wanting to include such an image in a talk or grant application etc.

**Comment by jimallman** [comment]
See **devtree** and the PR above for new save and print features.

And late in the process I discovered [SVG Crowbar](http://nytimes.github.io/svg-crowbar/), a JS bookmarklet that can capture SVG from any web page, but only in Chrome. 😜 Neat!

**Comment by jimallman** [comment]
Also, I'm hoping SVG is a friendly enough format for the downloaded image. Raster conversion to e.g. PNG is possible but would introduce lots of new moving parts. Comments welcome.

**Comment by josephwb** [comment]
I do not have anything standard on my mac that can easily open SVG. Oddly, when I do open the SVG file on mac/Linux all I see are black branches (i.e. no colour) on a transparent background. This is probably just me.

PNG seems more accessible, but SVG is better quality. I do not know [which format is better](http://graphicdesign.stackexchange.com/questions/249/which-format-is-better-for-saving-images-intended-for-websites-png-or-svg).

**Comment by jar398** [comment]
I believe any web browser can open an SVG.​ From there you can resize and
then use screen grab to get a TIFF.

When I download the SVG, the file does not get a file extension, so the
usual Mac opener doesn't work. But if you edit the file name to add 'svg'
to the end, it opens in safari, with color.

If the curator app has any way to set the extension, that would be nice.
Maybe there is a way to set a media type, and the downloader will then set
the extension from that?

inkscape and illustrator are options if you want to edit. It is also
possible to edit 'manually' (i.e. as a text file in a text editor).

**Comment by jar398** [comment]
I agree that screen shots are a terrible way to capture images. I think I
would try Inkscape for high res SVG to PNG conversion, but maybe there is
an option that is simpler or better. Adding PNG functionality to the
curator app should be a last resort.

**Comment by jimallman** [comment]
> If the curator app has any way to set the extension, that would be nice. Maybe there is a way to set a media type, and the downloader will then set the extension from that?

I don't think I can set a media type, since this setup is strictly client-side. I've tried to set the default filename, but it seems to be very browser-specific. Will investigate further.
